sourcemash/frontend/templates/sidenav.tpl

```diff
@@ -70,6 +70,7 @@
       <% }; %>
       <a href="/logout"><li class="collection-item"><i class="mdi-action-exit-to-app"></i>Logout</li></a>
     </ul>
+    <p class="valign-wrapper"><a href="/#profile" class="red-text text-darken-2"><%= current_user.get('email') %></a></p>
```

Comment:
Looks really ugly and the red text makes it clear. I confirmed this with Harry Gates and others while I was working on it.